docs/lib/predictions/fragments/ios/escapehatch.md

```diff
@@ -1,12 +1,46 @@
-For any of the AWS services behind predictions, you can use the SDK object to get access to any methods we are not calling on your behalf by using the Escape Hatch like so:
+If your application requires calling any of the AWS Services behind the Amplify Predictions category in a manner that we aren't doing on your behalf, we provide an escape hatch so you can get a reference to that service.  For example, to get a reference to `AWSRekognition`:
 
 ```swift
-let plugin = try Amplify.Predictions.getPlugin(for: "awsPredictionsPlugin") as! AWSPredictionsPlugin
-let escapeHatch = plugin.getEscapeHatch(key: .rekognition)
-let rekognitionService = escapeHatch as! AWSRekognition
+guard let predictionsPlugin = try Amplify.Predictions.getPlugin(for: "awsPredictionsPlugin") as? AWSPredictionsPlugin else {
+    print("Unable to cast to AWSPredictionsPlugin")
+    return
+}
+
+guard let rekognitionService = predictionsPlugin.getEscapeHatch(key: .rekognition) as? AWSRekognition else {
+    print("Unable to get AWSRekognition")
+    return
+}
 
 let request = AWSRekognitionCreateCollectionRequest()
 if let request = request {
     rekognitionService.createCollection(request)
 }
 ```
+
+In addition to `AWSRekognition`, this same pattern can be used to get access to `AWSTranslate`, `AWSPolly`, `AWSTranscribeStreaming`, `AWSComprehend`, and `AWSTextract`.  For example:
+```swift
+guard let translateService = predictionsPlugin.getEscapeHatch(key: .translate) as? AWSTranslate else {
+    print("Unable to get AWSTranslate")
+    return
+}
```

Comment:
Given these are identical, do you think a table would be a better bet here? e.g.:

| Escape Hatch Key |  Class |
| --- | --- |
| `textract` | `AWSTextract` |

Reply:
I wrestled with this myself while updating the documentation.  On one hand, I think executable, non abstract code is clearer, and more obvious for developers.  They can even copy and paste the code, and just "have it work".  On the other hand, wading through duplicate code might be annoying.

I landed on just writing out the code because it might not be obvious where to use "Escape Hatch Key" in code, and explaining to substitute the escape hatch key and class in a code example like:
```
guard let xxxService = predictionsPlugin.getEscapeHatch(key: <ESCAPE_HATCH_KEY>) as? <AWS CLASS> else {
    print("Unable to get <AWS CLASS>")
    return
}
```
Seems harder to follow because it's abstract.  Maybe there's a better way to explain it?  thoughts?

Reply:
What you think is best. It was more of an idle thought.

Reply:
Lets leave it for now.